keyboards/tada68/keymaps/peippo/keymap.c

```diff
@@ -0,0 +1,52 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	/* Default layer
+	* ,----------------------------------------------------------------------------------------------------------------------------.
+	* | GrvEsc|   1   |   2   |   3   |   4   |   5   |   6   |   7   |   8   |   9   |   0   |   +   |   ´   | Backspace  | MacOS |
+	* |----------------------------------------------------------------------------------------------------------------------------|
+	* | Tab      |   Q   |   W   |   E   |   R   |   T   |   Y   |   U   |   I   |   O   |   P   |   Å   |   ¨   |  Enter  | Del   |
+	* |-----------------------------------------------------------------------------------------------------------,        |-------|
+	* | Mod Layer |   A   |   S   |   D   |   F   |   G   |   H   |   J   |   K   |   L   |   Ö   |   Ä   |   '   |        | PgUp  |
+	* |----------------------------------------------------------------------------------------------------------------------------|
+	* | Shift   |   <  |   Z   |   X   |   C   |   V   |   B   |   N   |   M   |   ,   |   .   |   -   |   Shift   |  Up   | PgDn  |
+	* |----------------------------------------------------------------------------------------------------------------------------|
+	* | Control | Option | Command |                         Space                 | Cmnd  | Opt   | ModLr | Left  | Down  | Right |
+	* `----------------------------------------------------------------------------------------------------------------------------'
+	*/
+
+	[_BL] = LAYOUT_iso(
+		KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, AG_NORM,  \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
Ok, thanks for letting me know, I removed them.